tests/main/snap-run-devmode-classic/task.yaml

```diff
@@ -19,10 +19,11 @@ systems:
     # when the host has cgroupsv2 in it
     - ubuntu-16.04-*
 
-environment:
```

Comment:
Comment for the whole file: this change looks like something that belongs in a separate patch with a description that says what's going on.

Reply:
The problem is that we get a commit in a state that fails... I usually do not like to do that.